composer.json

```diff
@@ -22,7 +22,7 @@
         "drupal/honeypot": "^2.1",
         "drush/drush": "^12",
         "openeuropa/code-review": "^2.0.0-alpha6",
```

Comment:
the code review components are ready for php8.3 so we can upgrade these as well and check what changes. This one here https://github.com/openeuropa/code-review-drupal
Once that is done maybe the same fix applies for `SanitizeCommands::SANITIZE` as we did in oe_authentication (needs to be checked)